cmd/argo/commands/resubmit.go

```diff
@@ -24,6 +25,10 @@ func NewResubmitCommand() *cobra.Command {
   argo resubmit @latest
 `,
 		Run: func(cmd *cobra.Command, args []string) {
+			if cmd.Flag("priority").Changed {
+				cliSubmitOpts.priority = &priority
+			}
```

Comment:
We don't need this here if we use `priority *int32` instead of `priority int32`. Take a look at `commands/submit.go`.

Can you refactor this to match?

Reply:
Thanks. Just updated.